courses/models.py

```diff
@@ -17,15 +17,40 @@ def __str__(self):
 
 class Course(models.Model):
     """
-    An individual course within a Program, e.g. "Supply Chain 101"
+    A logical representation of a course, such as "Supply Chain Management
+    101". This won't have associated dates or any specific information about a
+    given course instance (aka course run), but rather only the things that are
+    general across multiple course runs.
+    """
+    program = models.ForeignKey(Program)
+
+    # These fields will likely make their way into the CMS at some point.
+    title = models.CharField(max_length=255)
+    thumbnail = models.ImageField(null=True, blank=True)
+    description = models.TextField(blank=True, null=True)
+    prerequisites = models.TextField(blank=True, null=True)
+
+    def __str__(self):
+        return self.title
+
+
+class CourseRun(models.Model):
+    """
+    An individual run of a course within a Program, e.g. "Supply Chain 101
+    - Summer 2017". This is different than the logical notion of a course, but
+      rather a specific instance of that course being taught.
     """
     title = models.CharField(max_length=255)
     edx_course_key = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
I'm debating whether some help text may be useful here. 

`Can be null, a deprecated course key or a split mongo course key. Intended for course keys on edx.org`